test/unit/transform-content-internals.test.js

```diff
@@ -14,6 +14,16 @@ describe('Transformed content passed to html parser is of correct type', () => {
 
     transformContent(inputObj, options)
 
-    expect(parseToHTML).toHaveBeenCalledWith(expect.any(String))
+    expect(parseToHTML).toHaveBeenCalledWith(expect.any(String), expect.anything())
   })
 })
+
+it('Passes the options to the html parser', () => {
```

Comment:
I don't really like `it` blocks with no `describe` but it's tricky to add a `describe` that matches the pattern already used in this file for this particular test.

Reply:
This seems fine by me my MO these days is to usually have a big ol' `describe` Wrapper around the whole unit, which would fit in any 'loose' `it`s